exercises/sublist/src/Sublist.hs

```diff
@@ -1,3 +1,6 @@
 module Sublist (Sublist(..), sublist) where
 
+data Sublist = Equal | Sublist | Superlist | Unequal deriving (Eq, Show)
```

Comment:
Agreed!

I'm usually in favor of anything to avoid the students having to read the test file.

Haskell is already hard enough to learn, and it is better to allow the students to see a lot of data declarations until they get used to it than let them stuck reading the test suites.